### PR TITLE
style(framework): resolve some lint warnings

### PR DESCRIPTION
**Pull request by alesmit**

### CODE REVIEW

File: packages/framework/src/decorators/cacheable.test.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ test('should set and read from cache', async () => {
     testMethodCalls = 0;
 
     @Cacheable({ key: 'key' })
-    async testMethod() {
+    async testMethod(): Promise<string> {
```
Comment (alesmit) [inline packages/framework/src/decorators/cacheable.test.ts:39]: test

Comment (alesmit) [reply packages/framework/src/decorators/cacheable.test.ts:39]: comment

**Comment by ukaoskid** [comment]
@alesmit Test comment from Telegram

File: packages/framework/src/decorators/cacheable.test.ts (hunk 1 above)
Comment (alesmit) [reply packages/framework/src/decorators/cacheable.test.ts:39]: @ukaoskid trying here

**Pull request merged**